### DISCUSSION TITLE
Pseudonymisierung in REDCap

### DISCUSSION THREAD

**Discussion by joundso** [REDCap-German-User-Group/Diskussion-FAQ#3] [Fragen und Antworten]
# Problem
In REDCap dürfen keine Patientenidentifikatoren wie die Patientennummer gespeichert werden. 

# Ziel
Erfahrungsaustausch, dazu, welches Vorgehen beim Erfassen von Daten, die pseudonymisiert in REDCap abgelegt werden sollen, am sinnvollsten wird.
Wo sollte die Pseudonymabbildung am sinnvollsten abgelegt werden?
Bestenfalls kann das auch standortübergreifend genutzt werden. Records in REDCap bekommen Suffixe pro Standort und werden durchnummeriert.

# Bekanntes
- Mainzelliste inkl. REDCap Plugin (Ist OK, aber die Nutzer können die Pseudonymabbildung nicht sehen. Pseudonymschema mit Prefix pro Standort ist nicht machbar)
- Excel-Liste (das wollen wir vermeiden)

**Comment by joundso** [comment]
Geht vlt. auch in Richtung https://github.com/REDCap-German-User-Group/Treuhandstelle/issues/1

**Reply by frissner** [reply]
Mittels des Plugin "Custom Record Auto-numbering - v1.0.5" ist ein DAG-spezifisches Prefix in gewissen Grenzen möglich.
siehe https://www.ctsi.ufl.edu/wordpress/files/2021/04/Custom-Record-Auto-numbering-External-Module-User-Guide.pdf 
Es funktioniert jedoch nur über die GUI. Das Prefix ersetzt die DAG-Nummer beim ersten Speichern eines CRF.
Es funktioniert NICHT beim Datenimport via CSV.
